e2e/rbd_helper.go

```diff
@@ -1094,6 +1104,31 @@ func getImageInfo(f *framework.Framework, imageName, poolName string) (imageInfo
 	return imgInfo, nil
 }
 
+// getImageStatus queries rbd about the given image and returns its metadata, and returns
+// error if provided image is not found.
+func getImageStatus(f *framework.Framework, imageName, poolName string) (imageStatus, error) {
+	// rbd --format=json status [image-spec | snap-spec]
+	var imgStatus imageStatus
+
+	stdOut, stdErr, err := execCommandInToolBoxPod(
+		f,
+		fmt.Sprintf("rbd status %s %s --format json", rbdOptions(poolName), imageName),
+		rookNamespace)
+	if err != nil {
+		return imgStatus, fmt.Errorf("failed to get rbd status: %w", err)
```

Comment:
It helps to give this `err != nil` and the below `stdErr != ""` errors a slightly different message. When something goes wrong, it can be identified more precisely if the error message is unique.

Reply:
done